Adyen Doc Update (#5870)
vuestorefront/vue-storefront: packages/core/docs/integrations/adyen.md

If the transaction is authorized, the server's controller for `payAndOrder`/`submitAdditionalPaymentDetails` will place an order in Commercetools and apply the `order` object to the response. Thanks to that, we have only one request from the client to both finalize/authorize a payment and make an order.

## Checkout.com
Adyen's module isn't compatible with [Checkout.com's module](https://github.com/vuestorefront/checkout-com).

## FAQ
### Error: NotFound: URI not found: /<project_name>/carts/<cart_id>

> `ctApi.apiHost` property inside your `middleware.config.js` contains unnecessary path. It has to be `https://<SHOP_DOMAIN>.com/` instead of `https://<SHOP_DOMAIN>.com/<project_name>/graphql`

Reviewer: ```suggestion
`ctApi.apiHost` property inside your `middleware.config.js` contains wrong path. It should be `https://<SHOP_DOMAIN>.com/` instead of `https://<SHOP_DOMAIN>.com/<project_name>/graphql`
```

### Error: The type with the key 'ctp-adyen-integration-web-components-payment-type' was not found

> You have to add new types and extension do commercetools as described [there](https://github.com/commercetools/commercetools-adyen-integration). We've pointed it out in 6th step of the Adyen's installation guide.

Reviewer: ```suggestion
You have to add new types and extensions to commercetools as described [on this page](https://github.com/commercetools/commercetools-adyen-integration). We've pointed it out in the 6th step of Adyen's installation guide.
```

Are you sure this link is correct? Also, add a link to installation guide mentioned (`integrations/adyen`)

Author: I decided to change the link to 2 links which point directly to HowToRun.md in both modules

## Checkout.com
Adyen's module isn't compatible with [Checkout.com's module](https://github.com/vuestorefront/checkout-com).

## FAQ
### Error: NotFound: URI not found: /<project_name>/carts/<cart_id>
`ctApi.apiHost` property inside your `middleware.config.js` contains wrong path. It should be `https://<SHOP_DOMAIN>.com/` instead of `https://<SHOP_DOMAIN>.com/<project_name>/graphql`

### Error: The type with the key 'ctp-adyen-integration-web-components-payment-type' was not found
You have to add new types and extension to commercetools as described on these pages:
- [Extension Module](https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/docs/HowToRun.md#commercetools-project-requirements),
- [Notification Module](https://github.com/commercetools/commercetools-adyen-integration/blob/master/notification/docs/HowToRun.md#commercetools-project-requirements).   

> We've pointed it out in 6th step of the [Adyen's installation guide](./adyen.html#installation).

Reviewer: ```suggestion
For more information, see the 6th step of the [Adyen's installation guide](./adyen.html#installation).
```

IMHO `We've pointed it out in 6th step` sounds like we blame the user for not reading the installation guide carefully :)
